src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let numberOfCalls = 0;
+
+  return (numberOne = 0) => {
+    return (numberTwo = 0) => {
+      numberOfCalls++;
+
+      const numbersSum = numberOne + numberTwo;
+
+      return numberOfCalls > 3 && numberOfCalls % 2 === 0
+        ? `Bzzz... Error!`
+        : numbersSum;
```

Comment:
Don't nest ternary operators. Write one, you can do this, we believe in you! Consider refactoring this ternary operation to improve readability.